subprojects/resources-http/src/main/java/org/gradle/internal/resource/transport/http/HttpClientConfigurer.java

```diff
@@ -159,6 +169,32 @@ public void configureUserAgent(HttpClientBuilder builder) {
         builder.setUserAgent(UriTextResource.getUserAgentString());
     }
 
+    private void configureCookieSpecRegistry(HttpClientBuilder builder) {
+        PublicSuffixMatcher publicSuffixMatcher = PublicSuffixMatcherLoader.getDefault();
```

Comment:
I don't see the need for doing this. `HttpClientBuilder` already initializes this with this value when unspecified.